### PR TITLE
Bump minor 0.2.0

### PR DESCRIPTION
**Pull request by sugargoat**

### CODE REVIEW

File: attest/core/Cargo.toml [hunk 1]
```diff
@@ -62,4 +62,4 @@ mbedtls-sys-auto = { git = "https://github.com/mobilecoinofficial/rust-mbedtls.g
 rand_core = { version = "0.5", default-features = false }
 chrono = "0.4"
 mc-crypto-rand = { path = "../../crypto/rand" }
-mc-sgx-build =  { version = "0.1.5-alpha-1", path = "../../sgx/build" }
+mc-sgx-build =  { version = "0.2.0", path = "../../sgx/build" }
```
Comment (jcape) [inline attest/core/Cargo.toml:65]: Let's remove the version entirely here.

File: attest/net/Cargo.toml [hunk 2]
```diff
@@ -34,4 +34,4 @@ sha2 = "0.8"
 rand = "0.7"
 
 [build-dependencies]
-mc-sgx-build =  { version = "0.1.5-alpha-1", path = "../../sgx/build" }
+mc-sgx-build =  { version = "0.2.0", path = "../../sgx/build" }
```
Comment (jcape) [inline attest/net/Cargo.toml:37]:
```suggestion
mc-sgx-build =  { path = "../../sgx/build" }
```

File: sgx/sync/Cargo.toml [hunk 3]
```diff
@@ -1,12 +1,12 @@
 [package]
 name = "mc-sgx-sync"
-version = "0.1.5-alpha-1"
+version = "0.2.0"
 authors = ["MobileCoin"]
 
 [dependencies]
-mc-sgx-libc-types = { version= "0.1.5-alpha-1", path = "../libc-types" }
+mc-sgx-libc-types = { version= "0.2.0", path = "../libc-types" }
```
Comment (jcape) [inline sgx/sync/Cargo.toml:7]:
```suggestion
mc-sgx-libc-types = { path = "../libc-types" }
```

File: sgx/sync/Cargo.toml [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 mc-sgx-types = { path = "../types" }
-mc-sgx-panic = { verson = "0.1.5-alpha-1", path = "../panic", optional = true }
+mc-sgx-panic = { verson = "0.2.0", path = "../panic", optional = true }
```
Comment (jcape) [inline sgx/sync/Cargo.toml:9]:
```suggestion
mc-sgx-panic = { path = "../panic", optional = true }
```

File: attest/core/Cargo.toml (hunk 1 above)
Comment (sugargoat) [reply attest/core/Cargo.toml]: Done!

**Pull request merged**